sugaroid/brain/oneword.py

```diff
@@ -29,14 +29,15 @@
 from nltk import word_tokenize, pos_tag
 
 from sugaroid.version import VERSION
-from sugaroid.brain.constants import BYE, ANNOYED, DISCLAIMER
+from sugaroid.brain.constants import BYE, ANNOYED, DISCLAIMER, HI_WORDS, HI_RESPONSES
 from sugaroid.brain.myname import MyNameAdapter
 from sugaroid.brain.ooo import Emotion
 from sugaroid.brain.postprocessor import random_response
 from sugaroid.brain.preprocessors import normalize
 from sugaroid.sugaroid import SugaroidStatement
 
 
+
```

Comment:
This looks like a redundant addition. It would be better to remove the newline addition